inference-engine/tests/functional/plugin/cpu/single_layer_tests/topk.cpp

```diff
@@ -0,0 +1,188 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "functional_test_utils/ov_tensor_utils.hpp"
+#include "common_test_utils/data_utils.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "ngraph_functions/builders.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+
+namespace CPULayerTestsDefinitions {
+using topKShape = std::pair<std::vector<ngraph::PartialShape>, std::vector<std::vector<ngraph::Shape>>>;
```

Comment:
use `InputShape`
see: https://github.com/openvinotoolkit/openvino/pull/8419/files#diff-9e062eac1d9f8d8b882557db6fc82d3fbb266206d7cebc3126c7fd4f9f53f1b2R15

Reply:
Done, thank you!